tomcat/src/main/java/org/apache/coyote/controller/LoginController.java

```diff
@@ -0,0 +1,63 @@
+package org.apache.coyote.controller;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import nextstep.jwp.db.InMemoryUserRepository;
+import nextstep.jwp.model.User;
+import org.apache.catalina.Session;
+import org.apache.coyote.controller.utils.PathFinder;
+import org.apache.coyote.http11.httpmessage.common.ContentType;
+import org.apache.coyote.http11.httpmessage.request.Request;
+import org.apache.coyote.http11.httpmessage.request.requestbody.RequestBodyContent;
+import org.apache.coyote.http11.httpmessage.response.Response;
+
+public class LoginController extends AbstractController {
+
+    @Override
+    public void service(Request request, Response response) throws Exception {
+        if (request.isPostMethod()) {
+            doPost(request, response);
+            return;
+        }
+        if (request.isGetMethod()) {
+            doGet(request, response);
+        }
+    }
+
+    @Override
+    protected void doPost(Request request, Response response) throws Exception {
+        final RequestBodyContent userInput = RequestBodyContent.parse(request.getBody());
+        final String account = userInput.getValue("account");
+        final String password = userInput.getValue("password");
+
+        if (InMemoryUserRepository.exist(account, password)) {
+            final User user = InMemoryUserRepository.findByAccount(account).get();
+            log.info("존재하는 유저입니다. ::: " + user);
+
+            if (request.hasHeader("Cookie")) {
+                response.redirect("/index.html")
+                        .addHeader("Content-Type", ContentType.HTML.getContentType() + ";charset=utf-8");
+                return;
+            }
+            final Session session = request.getSession(!request.hasHeader("Cookie"));
```

Comment:
😅음 전 이부분이 개인적으로 잘 읽히지 않더라구요 ㅠ

```suggestion
            final Session session = request.getSession(false);
```

Cookie 유무에 따른 분기처리인 것 같은데

이렇게 고쳐보는 건 괜찮은가요?? ㅎㅎ

Reply:
맞네요! 항상 Cookie가 헤더에 없는 경우만 해당 로직이 실행되니까 false로 의미를 명확하게 하는게 좋겠군요. 반영했습니다 😄

Reply:
ㅎㅎ 넵 !

Reply:
ㅎㅎ 넵 !